README.md

```diff
@@ -1,9 +1,23 @@
+
 # buyers-remorse-app
-A React based web application to increase personal awareness about buying choices.
+
+A React based web application that encourages better spending and buying habits, to reduce unwanted waste and regret. The application aims to achieve this by creating a scenario where the user is asked to reflect on why they want the item, and to wait a period of time of their choosing before being able to buy. The application also acts as a wish list of desired items.
+
 The running application can be found on [GitHub Pages](https://agiledev3.github.io/buyers-remorse-app)
 
```

Comment:
I would honestly remove all below this line.  

1) Prerequisites and installation - whoever does the development knows this stuff
2) Autogenerated from Available Scripts and down - it autogenerated so common for all projects - common knowledge, not interesting 
3) Citation - I don't feel it is relevant or gonna be ever used; it also feels like a part  of licensing but makes our MIT license more restrictive so there is some contradiction here
4) Contact - we have contact in the github ogranization info, it is usually the place for all the contacts

All other parts look good, imho.

Reply:
Let's just merge it.